composer.json

```diff
@@ -17,11 +17,12 @@
     ],
     "require": {
         "php": "^7.1 || ^8.0",
-        "doctrine/orm": "^2.3",
+        "doctrine/orm": "^2.3 || ^3.0",
         "symfony/deprecation-contracts": "^2.1 || ^3.0"
     },
     "require-dev": {
         "doctrine/doctrine-bundle": "^1.6 || ^2.0",
+        "friendsofphp/php-cs-fixer": "3.3.0",
```

Comment:
Don't add it as dependency
You can use this instruction to setup your local environment with predefined PHP-CS-Fixer
https://gist.github.com/adrenalinkin/35176a7b52b996666c4a36384fd536ad